client/components/buttons/UpdateButton.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+
+const propTypes = {
+  history: PropTypes.objectOf(PropTypes.any).isRequired,
+  recipe: PropTypes.objectOf(PropTypes.any).isRequired
+};
+
+class UpdateButton extends Component {
```

Comment:
This should be rewritten to be a stateless functional component seeing as it maintains no state of it's own nor does it use any of the lifecycle methods